src/Widgets/PaymentPlans/tests/Basics.tsx

```diff
@@ -0,0 +1,29 @@
+import { screen, waitFor, fireEvent } from '@testing-library/react'
```

Comment:
Splitting tests seems like a good idea. 
- by domain and functionalities
But I dont think removing `.test.tsx` prefix add value, on contrary.
- It a step toward a very unconventional naming pattern in js app development (every repo I see are using `.spec.tsx` or `test.tsx`)
- naming a test file with the same name as component  (`MyComponent.ts` is going to be tested by `MyComponent.ts`) and then wanting to import that component might lead to auto-importing tests instead of component.

Reply:
+ I would recommend using `__tests__/` folder  as it match the default jest pattern and also is close to the `__mocks__` folder that is likely to exist at some point in the application.

Reply:
You're right. It's fixed.